tests/app/dao/test_service_sms_sender_dao.py

```diff
@@ -155,6 +155,24 @@ def test_dao_add_sms_sender_for_service(self, notify_db_session, sample_provider
 
         assert new_sms_sender in service_sms_senders_after_updates
 
+    def test_dao_add_sms_sender_256_char(self, notify_db_session, sample_provider, sample_service) -> None:
+        provider = sample_provider()
+        service = sample_service()
+
+        new_sms_sender = dao_add_sms_sender_for_service(
+            service_id=service.id,
+            sms_sender='256Char - Its a dangerous business, Frodo, going out your door. You step onto the road, and if you dont keep your feet, theres no knowing where you might be swept off to. - One Ring to rule them all, One Ring to find them, One Ring to bring them all and in',
```

Comment:
Not that we don't all love LotR, but his is more concise . . .
```suggestion
            sms_sender=('a' * 256),
```
I don't insist on changing this.